### PR TITLE
CF-600 / CF-601: Several improvements

### PR DESCRIPTION
**Pull request by cschroer**
- Exclude new role for service agent from authoritative handling
- Add new network features
    - Support for secondary ranges (GKE)
    - New EU regions
    - Support for proxy only subnetworks

### CODE REVIEW

File: docs/DEFAULT-VPC.md [hunk 1]
```diff
@@ -15,58 +30,205 @@
...
+
+## IP configurations
+
+### RFC1918 free to use IP address ranges in the default VPC
+
+#### 10.0.0.0 - 10.255.255.255  (10/8 prefix)
+
+We will not use `10.192.0.0/10` for any further implementation. If you plan
+to add additional network features to the default VPC use any IP address
+range from here.
```
Comment (cova-fe) [inline docs/DEFAULT-VPC.md:66]: Isn't this a bit confusing when considered together with what is stated in line 37 and following?

Comment (cschroer) [reply docs/DEFAULT-VPC.md]: Changed that part a bit, is it more clear this way?

Comment (cova-fe) [reply docs/DEFAULT-VPC.md]: yup

**Review by cova-fe — APPROVED** [review]
lgtm

**Pull request merged**